src/main/java/servlets/CalcServlet.java

```diff
@@ -0,0 +1,31 @@
+package servlets;
+
+
+import controllers.Controller;
+import controllers.ServletController;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+@WebServlet(urlPatterns = "/calc/*")
+public class CalcServlet extends HttpServlet {
+    private static final Controller servletController = new ServletController();
+
+    @Override
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) {
+        servletController.getResult(request, response);
```

Comment:
I think we should wrap all controller calls in servlet in following way:
```
 catch (Exception e) {
            LOGGER.error("Something goes wrong", e);
            response.setStatus(HttpServletResponse.SC_INTERNAL_SERVER_ERROR);
        }
```

Because something bad can happened with inner code and user will see something unreadable and logs will empty.